helix-core/src/main/java/org/apache/helix/task/TaskDriver.java

```diff
@@ -1108,11 +1109,12 @@ public JobConfig getJobConfig(String job) {
     return TaskUtil.getJobConfig(_accessor, job);
   }
 
-  public JobContext getJobContext(String job) {
+  public JobContext getJobContext(String job) throws InvalidParameterException {
```

Comment:
Does it require all the current users to change their code?

Reply:
Yea good point. I think probably I should not add this in signature.